jest.config.ts

```diff
@@ -0,0 +1,88 @@
+import type {Config} from "jest";
+import os from "node:os";
+import pathPosix from "node:path/posix";
+
+
+let PRIMARY_REPORTER: string | [string, Record<string, unknown>] = "default";
+if ("undefined" !== typeof process.env.CI) {
+    PRIMARY_REPORTER = [
+        "github-actions",
+        {silent: false},
+    ];
```

Comment:
Before setting it to github-actions, shouldn't we check for the `GITHUB_ACTIONS` env var?